drivers/ethernet/Kconfig.e1000

```diff
@@ -0,0 +1,28 @@
+# Kconfig - Intel(R) PRO/1000 Gigabit Ethernet driver configuration options
+
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ETH_E1000
+	bool
+	prompt "Intel(R) PRO/1000 Gigabit Ethernet driver"
+	depends on NET_L2_ETHERNET PCI_ENUMERATION
+	help
+	  Enable Intel(R) PRO/1000 Gigabit Ethernet driver.
+
+config ETH_E1000_BASE_ADDRESS
+	hex "Intel(R) PRO/1000 Gigabit Ethernet Base Address"
+	depends on ETH_E1000
+	default 0XFEBC0000
```

Comment:
If we standardize on DTS, is this default still needed/useful?

Reply:
“depends on” is fixed. There’s nothing wrong with the macros here. Register names are mostly 3 characters long, using READ/WRITE would visually dominate over the register names.

Reply:
I agree w/Paul, we do we need to support non-DTS Kconfig?